modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -0,0 +1,262 @@
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OIDCCheckSessionEPUrl"),
+                "https://localhost:9853/oidc/checksession",
+                "Expected Session IFrame Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2IntrospectEPUrl"),
+                "https://localhost:9853/t/wso2.com/oauth2/introspect",
+                "Expected Token Introspection Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2RevokeEPUrl"),
+                "https://localhost:9853/oauth2/revoke",
+                "Expected Token Revocation Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OIDCLogoutEPUrl"),
+                "https://localhost:9853/oidc/logout",
+                "Expected Logout Endpoint URL not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2AuthzEPUrl"),
+                "https://localhost:9853/oauth2/authorize",
+                "Expected Authorization Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2DCREPUrl"),
+                "https://localhost:9853/t/wso2.com/api/identity/oauth2/dcr/v1.1/register",
+                "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OIDCDiscoveryEPUrl"),
+                "https://localhost:9853/t/wso2.com/oauth2/oidcdiscovery",
+                "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2JWKSPage"),
+                "https://localhost:9853/t/wso2.com/oauth2/jwks",
+                "Expected is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2TokenEPUrl"),
+                "https://localhost:9853/oauth2/token",
+                "Expected Token Endpoint URL is not found in the tenant mode");
+        Assert.assertEquals(fedAuthConfigTenantMap.get("OAuth2UserInfoEPUrl"),
+                "https://localhost:9853/oauth2/userinfo",
+                "Expected User Info Endpoint URL is not found in the tenant mode");
+
+        //Extract provisioning configurations for tenant.
+        ProvisioningConnectorConfig[] provisioningConfigsTenant = residentIdP.getProvisioningConnectorConfigs();
+
+        Map<String, String> provisioningConfigTenantMap = new HashMap<String, String>();
+        for (ProvisioningConnectorConfig config : provisioningConfigsTenant) {
+            for (Property property: config.getProvisioningProperties()) {
+                provisioningConfigTenantMap.put(property.getName(), property.getValue());
+            }
+        }
+
+        Assert.assertEquals(provisioningConfigTenantMap.get("scimGroupEndpoint"),
+                "https://localhost:9853/wso2/scim/Groups",
```

Comment:
This is not tenant supported?

Reply:
No. Only SCIM 2.0 endpoints have tenant domain in path.